example-react-app/package.json

```diff
@@ -3,12 +3,14 @@
   "private": true,
   "dependencies": {
     "@ant-design/icons": "^4.0.0",
-    "@haulmont/jmix-react-core": "file:../packages/jmix-react-core/haulmont-jmix-react-core-0.9.0-beta.1.tgz",
-    "@haulmont/jmix-react-ui": "file:../packages/jmix-react-ui/haulmont-jmix-react-ui-0.9.0-beta.1.tgz",
-    "@haulmont/jmix-rest": "file:../packages/jmix-rest/haulmont-jmix-rest-0.9.0-beta.1.tgz",
-    "@haulmont/react-ide-toolbox": "file:../packages/react-ide-toolbox/haulmont-react-ide-toolbox-0.1.0-beta.1.tgz",
+    "@apollo/client": "^3.3.6",
```

Comment:
I tend to think we should move libs we use into libs deps, otherwise it makes update process more complicated